src/App.js

```diff
@@ -1,26 +1,93 @@
-import React from 'react';
-import logo from './logo.svg';
+import React from "react";
+import Dropdown from "react-bootstrap/Dropdown";
+import BookList from "./book/BookList"
+import {
+    BrowserRouter as Router,
+    Link as Link,
+    Switch,
+    Route,
+} from "react-router-dom";
+
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+export default  function App() {
+    const levels = {
+        "lycée": [
+            {
+                "id": 287,
+                "name": "Terminale"
+            },
+            {
+                "id": 286,
+                "name": "1re"
+            },
+            {
+                "id": 285,
+                "name": "2de"
+            }
+        ],
+        "collège": [
+            {
+                "id": 284,
+                "name": "3ème"
+            },
+            {
+                "id": 282,
+                "name": "4ème"
+            },
+            {
+                "id": 281,
+                "name": "5ème"
+            },
+            {
+                "id": 283,
+                "name": "6ème"
+            }
+        ]
+    }
+
+    return (
+        <Router>
+            <header className="App App-header">
+                <Link to="/books" className="btn btn-basic">Accueil</Link>
+                {Object.keys(levels).map(key=>{
```

Comment:
To make it easier to understand, you can name the `key` variable something more application-specific, such as `school` (if I got that right).

Reply:
Pour la clarté du code, je n'y ai pas pensé